ROADMAP.md

```diff
@@ -15,22 +15,38 @@ Please open an issue to track any initiative on the roadmap of Contour (Usually
 The following table includes the current roadmap for Contour. If you have any questions or would like to contribute to Contour, please attend a [community meeting](https://projectcontour.io/community/) to discuss with our team. If you don't know where to start, we are always looking for contributors that will help us reduce technical, automation, and documentation debt.
 Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Contour.
 
-`Last Updated: August 2020`
+`Last Updated: February 2021`
+
+|Theme|Description|Timeline|Issue|Notes|
+|--|--|--|--|--|
+|Compatibility|Implement Kubernetes Ingress V1 specification (requires v1.16)|March 2021|[projectcontour/contour#2139](https://github.com/projectcontour/contour/issues/2139)||
+|Performance|Rate limiting support|February 2021|[projectcontour/contour#370](https://github.com/projectcontour/contour/issues/370)| Local rate limiting support arrived in 1.12 in January 2021, global rate limiting support is due in 1.13, February 2021.|
+|General|Wildcard Path Matching|March 2021|[projectcontour/contour#2138](https://github.com/projectcontour/contour/issues/2138)|A limited form of this is required for both Ingress and Gateway-APIs.|
+|Observability|Support Access Log Service|April 2021|projectcontour/contour#1291||
+|Gateway APIs|Introduce support for upstream Gateway APIs|April 2021|[projectcontour/contour#2287](https://github.com/projectcontour/contour/issues/2287)||
```

Comment:
I suggest linking to https://github.com/kubernetes-sigs/gateway-api for context.